packages/expressive-code/README.md

```diff
@@ -30,17 +30,21 @@ npm install expressive-code
 ## Usage example
 
 ```js
-import { ExpressiveCode, ExpressiveCodeConfig } from 'expressive-code'
+// File: generate-html.js
```

Comment:
```suggestion
// File: generate-html.mjs
```
As I just tried to run the code in a blank project, I got the error `Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.` Most users will probably have their `type` configured correctly, but to help the rest trying to run the example code, we could maybe use the `.mjs` extension here.

Reply:
lgtm!